crates/graphql-composition/src/federated_graph/directives/is.rs

```diff
@@ -0,0 +1,58 @@
+use cynic_parser_deser::{ConstDeserializer as _, ValueDeserialize};
```

Comment:
New file, we were using RequireDirective for `@is`, which works but is just weird.

Reply:
used a type alias for this in engine-schema